plugins/webp-uploads/settings.php

```diff
@@ -210,19 +199,70 @@ function webp_uploads_use_picture_element_callback(): void {
-			<?php esc_html_e( 'Use <picture> Element', 'webp-uploads' ); ?>
+			<input
+				type="checkbox"
+				id="webp_uploads_use_picture_element"
+				name="webp_uploads_use_picture_element"
+				aria-describedby="webp_uploads_use_picture_element_description"
+				value="1"
+				<?php checked( $picture_element_option ); // Option intentionally used instead of webp_uploads_is_picture_element_enabled() to persist when perflab_generate_webp_and_jpeg is updated. ?>
+				<?php disabled( ! $jpeg_fallback_enabled ); ?>
+			>
+			<?php
+			/*
+			 * If the checkbox is disabled, but the option is enabled, include a hidden input to continue sending the
+			 * same value upon form submission.
+			 */
+			if ( ! $jpeg_fallback_enabled && $picture_element_option ) {
+				?>
+				<input
+					type="hidden"
+					id="webp_uploads_use_picture_element_hidden"
+					name="webp_uploads_use_picture_element"
+					value="1"
+				>
+				<?php
+			}
+			esc_html_e( 'Use <picture> Element', 'webp-uploads' );
+			?>
 			<em><?php esc_html_e( '(experimental)', 'webp-uploads' ); ?></em>
 		</label>
 		<p class="description" id="webp_uploads_use_picture_element_description"><?php esc_html_e( 'The picture element serves a modern image format with a fallback to JPEG. Warning: Make sure you test your theme and plugins for compatibility. In particular, CSS selectors will not match images when using the child combinator (e.g. figure > img).', 'webp-uploads' ); ?></p>
-		<div id="webp_uploads_jpeg_fallback_notice" class="notice notice-info inline" <?php echo $picture_element_enabled ? '' : 'hidden'; ?>>
+		<div id="webp_uploads_jpeg_fallback_notice" class="notice notice-info inline" <?php echo $picture_element_option ? '' : 'hidden'; ?>>
 			<p><?php esc_html_e( 'Picture elements will only be used when JPEG fallback images are available. So this will not apply to any images you may have uploaded while the "Also generate JPEG" setting was disabled.', 'webp-uploads' ); ?></p>
 		</div>
 	</div>
 	<script>
 		document.getElementById( 'webp_uploads_use_picture_element' ).addEventListener( 'change', function () {
 			document.getElementById( 'webp_uploads_jpeg_fallback_notice' ).hidden = ! this.checked;
 		} );
+
+		// Listen for clicks on the JPEG output checkbox, enabling/disabling the
+		// picture element checkbox accordingly.
+		document.getElementById( 'perflab_generate_webp_and_jpeg' ).addEventListener( 'change', function () {
+			document.querySelector( '.webp-uploads-use-picture-element' ).classList.toggle( 'webp-uploads-disabled', ! this.checked );
+			document.getElementById( 'webp_uploads_picture_element_notice' ).hidden = this.checked;
+			document.getElementById( 'webp_uploads_picture_element_fieldset' ).classList.toggle( 'disabled', ! this.checked );
+
+			var checkbox = document.getElementById( 'webp_uploads_use_picture_element' );
```

Comment:
We can embrace `const` now. Also, PhpStorm flags the use of `var`. 

```suggestion
			const checkbox = document.getElementById( 'webp_uploads_use_picture_element' );
```